ckanext/datitrentinoit/model/mapping.py

```diff
@@ -170,6 +168,7 @@ def create_pro_package_dict(guid, swpentry: StatWebProEntry, metadata: StatWebMe
     extras['Fenomeno'] =  metadata.get_fenomeno()
     extras['Confronti territoriali'] = metadata.get_confronti()
     extras['_harvest_source'] = 'statistica:' + swpentry.get_id()
+    extras["identifier"] = sha1(f"statistica:{swpentry.get_id()}".encode()).hexdigest()
```

Comment:
The generation of identifier it's ok for the variable part, but now we're missing the IPA prefix.